### PR TITLE
docs: push notifications v2

### PR DESCRIPTION
**Pull request by vishalnarkhede**
Pending Items (which will be covered in separate PRs)

- Troubleshooting guide
- Updating Sample app with push notification

### CODE REVIEW

File: docusaurus/docs/reactnative/guides/push_notifications_v1.mdx [hunk 1]
```diff
@@ -1,14 +1,16 @@
 ---
-id: push-notifications
+id: push-notifications-v1
 sidebar_position: 1
```
Comment (ferhatelmas) [inline docusaurus/docs/reactnative/guides/push_notifications_v1.mdx:3]: does it require changing?

**Review by ferhatelmas — APPROVED** [review]
LGTM

File: docusaurus/docs/reactnative/guides/push_notifications_v1.mdx (hunk 1 above)
Comment (vishalnarkhede) [reply docusaurus/docs/reactnative/guides/push_notifications_v1.mdx:3]: the `id` you mean? Or label

Comment (ferhatelmas) [reply docusaurus/docs/reactnative/guides/push_notifications_v1.mdx:3]: `sidebar_position`

Comment (ferhatelmas) [reply docusaurus/docs/reactnative/guides/push_notifications_v1.mdx:3]: right now, index is same it fallbacks to label I guess

**Comment by mfbx9da4** [comment]
@vishalnarkhede 

Where is the documentation about how to configure the app bade count from the push notification data?

**Pull request merged**